Gitea shared service doc updates (#1684)
microsoft/AzureTRE: docs/tre-admins/setup-instructions/configuring-shared-services.md

## Configure Gitea repository

> Note : This is a shared Gitea for all workspace, you can also create a Gitea service per workspace (not created by default).

Reviewer: ```suggestion
Note : This is a Gitea *shared service* which will be accessible from all workspaces intended for mirroring external Git repositories. A Gitea *workspace service* can also be deployed per workspace to enable Gitea to be used within a specific workspace.
```

> By default, this Gitea is correctly configured but empty.

Reviewer: ```suggestion
By default, this Gitea instance does not have any repositories configured. You can add repositories to Gitea either by using the command line or by using the Web interface.
```

You can add repositories to Gitea either by command line or using the Web interface.

### By command line

1. On the jumbox, run:
```./scripts/gitea_migrate_repo.sh -t <tre_id> -g <URL_of_github_repo_to_migrate>```
1. If you have issues with token or token doesn't work, you can reset the token by setting it's value to null in Key Vault:
```az keyvault secret set --name gitea-<tre-id>-admin-token --vault-name kv-<tre-id> --value null```

### By the web interface

1. on the jumbox, open Edge and go to:
```https://gitea-<TRE_ID>.azurewebsites.net/```
1. Authenticate yourself using username ```giteaadmin``` and the secret ```<gitea-TRE_ID-administrator-password>``` stored in the keyvault,
1. Add the repository of your choice

> ### Check the result

Reviewer: ```suggestion
### Verify can access the mirrored repository
```

> In a workspace VM:
> 
> command line: ```git clone https://gitea-<TRE_ID>.azurewebsites.net/giteaadmin/<NameOfrepository>```
> 
> web URL: ```https://gitea-<TRE_ID>.azurewebsites.net/```

Reviewer: ```suggestion
From a virtual machine within a workspace:
- Command line: ```git clone https://gitea-<TRE_ID>.azurewebsites.net/giteaadmin/<NameOfrepository>```
- Gitea Web Interface: ```https://gitea-<TRE_ID>.azurewebsites.net/```
```
